CONTRIBUTING-development.md

```diff
@@ -0,0 +1,150 @@
+# Coding style
```

Comment:
nit: I think a better name for this file would be: `CONTRIBUTING-styleguide.md` (feel free to suggest an even better one)

Reply:
as discussed over chat. Let's go for STYLEGUIDE.md